packages/adaptive_scaffold/README.md

```diff
@@ -86,24 +125,26 @@ SlotLayout.from creates a SlotLayoutConfig holds the actual widget to be display
         config: <Breakpoint, SlotLayoutConfig>{
           Breakpoints.medium: SlotLayout.from(
             inAnimation: AdaptiveScaffold.leftOutIn,
-            key: const Key('pnav1'),
+            key: const Key('Primary Navigation'),
```

Comment:
This is aside from the actual line that I'm commenting on.

It might be useful if we make these examples collapsable. For this landing page README.md, it will be better if there's less content and more impact via pictures, etc. That way, users can window shop and see whether this package is fit for them.

Reply:
I don't think that's possible with the given excerpt mechanism, unfortunately.